tasks/connectors/edgescan/edgescan.rb

```diff
@@ -79,10 +79,12 @@ def run(opts)
         edgescan_api.fetch_in_batches do |edgescan_assets, edgescan_definitions|
           edgescan_assets.each do |edgescan_asset|
             kenna_api.add_assets(edgescan_asset)
-            if @options[:create_findings]
-              kenna_api.add_vulnerabilities(edgescan_asset.vulnerabilities)
-            else
-              kenna_api.add_findings(edgescan_asset.vulnerabilities)
+            if @options[:include_network_vulnerabilities] || @options[:include_application_vulnerabilities]
+              if @options[:create_findings]
```

Comment:
This conditional appears to be backwards - looks like a typo to me

Reply:
So this is saying that if you want to import any edgescan vulnerabilities (network, application, or both), do you want to create Kenna findings or Kenna vulnerabilities from them then.
I believe this is the desired functionality.

Reply:
Oh, this one:

```
if @options[:create_findings]
    kenna_api.add_vulnerabilities(edgescan_asset.vulnerabilities)
else
    kenna_api.add_findings(edgescan_asset.vulnerabilities)
```

This condition says to create vulnerabilities if the create findings flag is set and vice-versa.

Reply:
Fixed in [3d5efeb](https://github.com/KennaSecurity/toolkit/pull/403/commits/3d5efebc2754ca3214e37519754d284cf2da7ae9)